### PR TITLE
first stab at new primary documentation

### PR DESCRIPTION
**Pull request by xrotwang**
Things which need to be added:
- docs about how we want to use issues
- docs about editorial policy on ZENODO

What else?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,88 +2,75 @@
...
 
-Lexibank consists of several pieces:
-- The Lexibank dataset format, based on CLDF
-- The Lexibank workflow and software: `pylexibank`
-- The [Lexibank GitHub organisation](https://github.com/lexibank)
-- The [Lexibank ZENODO community](https://zenodo.org/communities/lexibank)
+Lexibank is a curated collection of lexical datasets.
+The Lexibank infrastructure consists of
+- a dataset specification based on [CLDF Wordlists](https://github.com/cldf/cldf/tree/master/modules/Wordlist)
+- a registry of published Lexibank datasets on [ZENODO](https://zenodo.org/communities/lexibank)
```
Comment (chrzyki) [inline README.md]: It should always be `Zenodo`, not `ZENODO` I think.

File: README.md [hunk 2]
```diff
@@ -2,88 +2,75 @@
...
-  ```
-- Almost always lexibank datasets refer to specific versions of Glottolog and Concepticon data, as indicated in [cldf-metadata.json](https://github.com/lexibank/bowernpny/blob/12e6d222593028696f871e409bb5b1175daad627/cldf/cldf-metadata.json#L16-L23). Care should be taken to only refer to released versions of these repositories for released versions of the dataset.
 
+## The Lexibank community on ZENODO
 
-## The Lexibank workflow and `pylexibank`
+GitHub is not a viable platform for longterm (or even midterm) preservation of Lexibank datasets. However, it provides an excellent collaborative curation platform, and can easily be hooked up with ZENODO, thereby providing longterm preservation for released versions of datasets.
 
-TODO
+To establish a "corporate identity" of Lexibank datasets on ZENODO, such datasets **should** be submitted to the [lexibank ZENODO community](https://zenodo.org/communities/lexibank). We also recommend the keywords "CLDF" for Lexibank datasets on ZENODO.
```
Comment (SimonGreenhill) [inline README.md]: rather than corporate identity,  it's more findability?

Comment (xrotwang) [reply README.md]: Ah, right. We should probably mention that Lexibank is
- Zenodo for **F**indable and **A**ccessible
- CLDF for **I**nteroperable
- Lexibank itself for **R**eusable.

=> Lexibank is FAIR lexical data :)

**Comment by LinguList** [comment]
shoudl we already merge this?

**Comment by xrotwang** [comment]
yes, I think so.

**Pull request merged**